src/models/stackedArea.js

```diff
@@ -97,7 +97,13 @@ nv.models.stackedArea = function() {
             gEnter.append('g').attr('class', 'nv-scatterWrap');
 
             wrap.attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
-
+            
+            // If the user has not specified forceY, make sure 0 is included in the domain
+            // Otherwise, use user-specified values for forceY
+            if (scatter.forceY().length == 0) {
+                scatter.forceY().push(0);
+            }
```

Comment:
You should be able to just do scatter._forceY([0])

Adding an underscore before an option means it's an internally-set option and that if a user overrides it themselves with forceY then it'll ignore the value from _forceY after that.